packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -950,6 +952,18 @@ export class DeltaManager
 
     private readonly opHandler = (documentId: string, messagesArg: ISequencedDocumentMessage[]) => {
         const messages = messagesArg instanceof Array ? messagesArg : [messagesArg];
+        const sequence = messages[0].sequenceNumber;
+        const last = messages[messages.length - 1].sequenceNumber;
+        if (this.lastSequenceFromSocket !== undefined && sequence !== this.lastSequenceFromSocket + 1) {
```

Comment:
@GaryWilber , this check assumes socket does not have any duplicate ops. You mentioned that might not be the case. Can you please glance and clarify?
@tanviraumi - FYI, we are trying to get to the bottom of issues we see in stress scalability tests where (I believe) we see gaps in op stream. We need (somewhere) to clearly describe protocol expectations, i.e. what client could rely on when it comes to ops on the socket (including initial ops).

Reply:
Example scenario:
1. Server broadcasts ops 10 to 20 to the client
2. Server then crashes before it checkpoints its state
3. Server restarts and loads from the previous state. It will broadcast ops 10 to 20 to the client a second time.

Reply:
I've added code to take it into account, but I wonder - shouldn't it result in closed connection and starting over?

Reply:
Thanks for explanation - I modified code to take it into account (I believe :))